app/views/couches/photos/new.html.haml

```diff
@@ -0,0 +1,15 @@
+= form_for @photo do |f|
+  = f.label :title
+  = f.text_field :title
+
+  = f.label :caption
+  = f.text_field :caption
+
```

Comment:
do we want to insert a caption that it only accepts filetypes jpg, jpeg, gif, png?